demo/customer-sdk-demo.html

```diff
@@ -89,8 +105,11 @@
         zoom: 8
       });
 
-      // beside
-      BringgSDK.connect(customer_access_token, onConnect);
+      // example for connection using order uuid and share uuid
+      //BringgSDK.connect(customer_access_token, onConnect);
```

Comment:
can leave this commented out but please see my comment above about name changing so you'll change the method name in the comment